docs/content/docs/concepts/summarization.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Summarization
+menuPosition: 10
+---
+
+<!-- IMPORTANT: Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path
+     argument) and then run "npm run build:md-magic" from the docs folder. That will update all autogenerated
+     content, so you should commit the resulting changes. -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE:path=../../../../packages/runtime/container-runtime/src/summary/README.md) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
+
+## Table of contents
+
+-   [Introduction](#introduction)
+    -   [Summary vs snapshot](#summary-vs-snapshot)
+-   [Why do we need summaries?](#why-do-we-need-summaries)
+-   [Who generates summaries?](#who-generates-summaries)
+-   [When are summaries generated?](#when-are-summaries-generated)
+-   [How are summaries generated?](#how-are-summaries-generated)
+    -   [Summary Lifecycle](#summary-lifecycle)
+    -   [Single-commit vs two-commit summaries](#single-commit-vs-two-commit-summaries)
+    -   [Incremental summaries](#incremental-summaries)
+    -   [Resiliency](#resiliency)
+-   [What does a summary look like?](#what-does-a-summary-look-like)
+
+## Introduction
+
+This document provides a conceptual overview of summarization without going into a lot of technical or implementation details. It describes what summaries are, how / when they are generated and what do they look like. The goal is for this to be an entry point into summarization for users and developers alike.
+
+### Summary vs snapshot
+
+The terms summary and snapshot are sometimes used interchangeably. Both represent the state of a container at a point in time. They differ in some respects which are described in [this FAQ](https://fluidframework.com/docs/faq/#summarization).
+
+## Why do we need summaries?
```

Comment:
The most important advice I can give you is to provide more of the "why" and the goals/purposes of summaries. Then the point of the detailed parts can be explained by saying how they contribute to one of the goals. For example, what goal of summaries is advanced by the whole ack/nack business? What goal is advanced by the broadcast of the summary?

Reply:
That's a good point. However, the thing is that the summary broadcast ops and acks have no purpose other than to contribute to the competion of the summary process. This lets summarizer clients keep track of the latest summary.

So, really, the only goal of summaries is to have the state of the container at a point in time for future clients to start at.